packages/dashboard-backend/src/workspace.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Red Hat, Inc.
```

Comment:
This is what we need to update, but right now, the linter expects:

```suggestion
 * Copyright (c) 2018-2021 Red Hat, Inc.
```